### PR TITLE
Log at lower level when installing bundles

### PR DESCRIPTION
**Pull request by hmusum**
@gjoranv please review
@kkraune FYI

### CODE REVIEW

**Comment by kkraune** [comment]
👍

**Review by baldersheim — APPROVED** [review]

File: container-core/src/main/java/com/yahoo/container/core/config/ApplicationBundleLoader.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public synchronized void useBundles(List<FileReference> newFileReferences) {
 
         bundlesFromNewGeneration = installBundles(newFileReferences);
         BundleStarter.startBundles(activeBundles.values());
-        log.info(installedBundlesMessage());
+        log.fine(installedBundlesMessage());
```
Comment (gjoranv) [inline container-core/src/main/java/com/yahoo/container/core/config/ApplicationBundleLoader.java:73]: Why don't we want to see this?

**Comment by hmusum** [comment]
For me it's just noise. I would guess it's useful if a customer has issues with bundle loading or we want to debug bundle loading issues? It's shown both for application containers and other containers (config server, cluster controller, metrics proxy), so if it's possible to log less for some types of services that could be a thing to look into.

If you think this we need this log message I could add it back, of course.

**Comment by gjoranv** [comment]
Yes, I'll add a Jira task to myself.

**Pull request merged**